packages/js/e2e-core-tests/specs/merchant/wp-admin-page-loads.test.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Internal dependencies
  */
-const { merchant } = require( '@woocommerce/e2e-utils' );
+const { merchant, waitForSelectorWithoutThrow } = require( '@woocommerce/e2e-utils' );
```

Comment:
Since `waitForSelectorWithoutThrow` isn't used in this file, I think we can remove the import.

Reply:
Nice catch. Fixed in https://github.com/woocommerce/woocommerce/pull/31498/commits/4ee75fc5d2f337acab41a5ca8020dc3883260ee1